.pipelines/pipeline.yaml

```diff
@@ -329,6 +329,17 @@ stages:
         vmSize: Standard_B2ms
         k8sVersion: ""
         dependsOn: "containerize"
+    # Azure Overlay E2E Stateless CNI tests
+    - template: singletenancy/azure-cni-overlay-stateless/azure-cni-overlay-stateless-e2e-job-template.yaml
+      parameters:
+        name: "azure_overlay_stateless_e2e"
+        displayName: Azure Stateless CNI Overlay
+        os: windows
+        clusterType: overlay-byocni-up
+        clusterName: "statelesswin"
+        vmSize: Standard_B2ms
+        k8sVersion: "1.30"
```

Comment:
Note: this line should/will be removed once #2924 is merged.

Reply:
done